pkg/translator/prometheusremotewrite/metrics_to_prw_v2.go

```diff
@@ -80,7 +80,12 @@ func (c *prometheusConverterV2) fromMetrics(md pmetric.Metrics, settings Setting
 					}
 					c.addGaugeNumberDataPoints(dataPoints, resource, settings, promName)
 				case pmetric.MetricTypeSum:
-					// TODO implement
+					dataPoints := metric.Sum().DataPoints()
```

Comment:
If the sum is non-monotonic, we need to emit it as a gauge instead.

Reply:
[13ee3b6](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37156/commits/13ee3b61e209fe3b1965a351e37a40b42c0b84e2)
Addressed